### PR TITLE
Gitignore acceptance test xunit output.

### PR DESCRIPTION
**Pull request by benpatterson**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,7 +40,7 @@ htmlcov
 .tox
 nosetests.xml
 unittests.xml
-coverage/
```
Comment (benpatterson) [inline .gitignore:43]: Author's note: this line a duplicate. See line 37.

**Comment by benpatterson** [comment]
@rlucioni @schenedx Missed this in #665

**Comment by schenedx** [comment]
:+1:

**Comment by rlucioni** [comment]
👍

**Comment by vkaracic** [comment]
:metal:

**Pull request merged**